build.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#!/usr/bin/env python2
```

Comment:
So we will still use python2 for running build script but use python3 for other config files?

Reply:
Ideally, I should just do that change too. Good catch!

Reply:
Updated with a new commit to change it to python3.